Brewfile

```diff
@@ -1,4 +1,9 @@
 brew 'python@2'
+brew 'yarn'
 brew 'libxmlsec1'
 brew 'openssl'
-brew 'chromedriver'
+brew 'redis', restart_service: true
+brew 'postgresql@9.4', restart_service: true, link: true
```

Comment:
According to our own [docs](https://docs.sentry.io/server/installation/), this should be postgresql@9.5.

Reply:
According to more of our own [docs](https://docs.sentry.io/internal/environment/), postgresql is 9.4. I'm going to assume that we run on 9.5 then?

Reply:
Fun fact, in production, we use 9.3. The differences here are relatively insignificant, and any of the above will be fine. I'd probably personally pick 9.6 since it's the latest in 9.x series.

Reply:
Currently Travis CI's postgres service is on 9.6.x, so it should be ok to pin to 9.6. Travis redis is 4.x, but brew doesn't have an explicit pin for it, so `redis@3.2` should be ok for now until we move towards a more platform-independent docker-based system setup to replace homebrew.